app/hotels/src/singleHotel/summary/RoomSummary.js

```diff
@@ -25,23 +26,16 @@ function RoomSummary(props: Props) {
   return (
     <View style={styles.container}>
       <View style={styles.content}>
-        <Text style={styles.summary}>
-          <Translation id="single_hotel.room_summary.summary" />
-        </Text>
-        <View style={styles.row}>
-          <Translation
-            id="single_hotel.book_now.description"
-            values={{
-              personCount: props.getPersonCount(),
-              numberOfRooms: props.numberOfRooms,
```

Comment:
Will that be shown back in a later PR?

Reply:
This is not in the new design. Check the figma link